projects/caddy/parser_fuzzer.go

```diff
@@ -0,0 +1,12 @@
+package caddyfile
+
+func testParser(input string) parser {
+	return parser{Dispenser: NewTestDispenser(input)}
+}
+
+func FuzzParser(data []byte) int {
+	p := testParser(string(data))
+	p.Next()
+	_ = p.parseOne()
+	return 1
+}
```

Comment:
We had to remove the Caddyfile parser fuzzer in caddyserver/caddy#3288 due to, what I'd call, unbounded glob expansion. Basically when Caddy expands a glob, and perhaps due to the nature of the CI env, the resulting slice of files is too big, which results in OOM. Granted, we called on the parser to parse the entire file rather than one step, as here.

@mholt and/or @francislavoie, I'm a bit rusty in the caddyfile parser codebase. Will `parseOne()` expand globs? Or will it just do one pass through the file first without expanding or importing anything?

Reply:
`parseOne` will eventually call `import` so it will expand globs; you can have files `1.txt` and `2.txt` then a Caddyfile like:
```
:80 {
	import *.txt
}
```
and it will import tokens from both of those files (seems to be in alphabetical order but idk how they end up being sorted in the end)

Reply:
So we will run into the same problem of OOM due to expansion leading to too many files loaded. It's probably best to drop this fuzzer due to false negatives.

Reply:
Ah yep, I remember this. I recommend dropping it unless a better solution is found.

Reply:
Parser fuzzer has been removed